src/opera/pge/base_pge.py

```diff
@@ -203,14 +206,40 @@ class and its inheritors, and as such, this class assumes access to the
     def __init__(self):
         self.name = None
         self.logger = None
+        self.runconfig_path = None
+        self.runconfig = RunConfig(self.runconfig_path)
+        self.catalog_metadata = {}
 
     def _run_sas_qa_executable(self):
         # TODO
         pass
 
     def _create_catalog_metadata(self):
-        # TODO
-        pass
+        """Returns the catalog metadata as a Python dictionary"""
+
+        self.cached_catalog_metadata = {
+            'PGE_Name': self.runconfig.pge_name,
+            'PGE_Version': "1.0.test",
```

Comment:
Would be good to get a ``#TODO`` here so we remember to revisit once we've integrated proper versioning

Reply:
Done